src/modules/PublicLab.MainImageModule.js

```diff
@@ -148,6 +148,18 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
 
     });
 
+    var input = document.getElementById( 'thumbnail-img' );
+    var infoArea = document.getElementById( 'thumbnail-filename' );
+
+    input.addEventListener( 'change', showFileName );
+
+    function showFileName( event ) {
```

Comment:
The style should be consistent with the rest of the file.

Reply:
Thanks for the review @VladimirMikulic. Solved.

Reply:
I don't see it solved?